.rubocop.yml

```diff
@@ -46,23 +48,42 @@ Style/DotPosition:
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# Maintain compatibility with older Rubies
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
 # Allow multiline block chains
 Style/MultilineBlockChain:
   Enabled: false
 
+# Indent one level for follow-up lines
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+# Indent one level for follow-up lines
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+# TODO: Discuss
+Style/MutableConstant:
+  Enabled: false
+
+# TODO: Discuss
+Style/ParallelAssignment:
+  Enabled: false
+
 # Allow Perl-style references to regex matches
 Style/PerlBackrefs:
   Enabled: false
 
-Style/Documentation:
-  Exclude:
-    - 'lib/reek/ast/sexp_extensions/send.rb'
-    - 'lib/reek/ast/sexp_extensions/super.rb'
-    - 'lib/reek/ast/sexp_extensions/variables.rb'
-
-Style/AccessorMethodName:
-  Exclude:
-    - 'lib/reek/context/visibility_tracker.rb'
+# Always use raise to signal exceptions
+Style/SignalException:
+  EnforcedStyle: only_raise
```

Comment:
>> Is there any place you would like to use fail

We haven't written any "fails" / "raises" ever since we introduced rubocop :laughing: 
But yes, grepping for "raise" shows me a couple of places where I would do that. 
On the other hand, that's the same discussion we recently had in [RubyCritic](https://github.com/whitesmith/rubycritic/pull/119) so let me just quote myself (ah, i love quoting myself!!!):

>> Not the biggest fan of this since I really like the semantic difference between "fail" and "raise". 
However, sticking a reasonable style guide project wide is more important than my personal taste here.

So let's just stick with rubocop's defaults ;)